html-block-and-inline/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>HTML Block and Inline</title>
+</head>
+<body>
+  <h1><b>Blue Americana Snuggie&trade; Blanket</b></h1>
```

Comment:
you don't need the `b` tags when using one of the `h` tags. they are bold by default.